src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -122,6 +123,21 @@ public bool ShowRawData
 
         public bool IsNodeExpanded { get; set; }
 
+        /// <summary>
+        /// If Content is 1 string, e.g. "Empty", "null", "_SingleFunctionObject", margin should be more to the left side.
+        /// For this purpose used this value. When it's true, margin in DataTrigger is set to -15,5,5,5; otherwise
+        /// it's set to 5,5,5,5
+        /// </summary>
+        public bool IsOneRowContent
```

Comment:
when you have such specific behavior, you can have a test written. Otherwise there will be regressions.
You can very well simulate the UI. Like place a codeblock with value and without value. Raise a mouse over and get the preview control, check the margin. It is not that tough. Try it once.